packages/components/src/progress-bar/styles.ts

```diff
@@ -38,6 +36,10 @@ export const Track = styled.div`
 	// Windows high contrast mode.
 	outline: 2px solid transparent;
 	outline-offset: 2px;
+
+	:where( & ) {
```

Comment:
@DaniGuardiola Is that what you had in mind when you suggested using `:where`? AFAIK, the main benefit is the zero specificity, right?